nemo/collections/asr/parts/utils/diarization_utils.py

```diff
@@ -15,500 +15,194 @@
 import copy
 import csv
 import json
-import math
 import os
+import shutil
+import sys
 from collections import OrderedDict as od
 from datetime import datetime
-from typing import List
+from typing import Dict, List, Tuple
 
-import librosa
 import numpy as np
-import soundfile as sf
-import torch
 
-from nemo.collections.asr.metrics.wer import WER
-from nemo.collections.asr.metrics.wer_bpe import WERBPE
-from nemo.collections.asr.models import ClusteringDiarizer, EncDecCTCModel, EncDecCTCModelBPE, EncDecRNNTBPEModel
+from nemo.collections.asr.metrics.wer import word_error_rate
+from nemo.collections.asr.models import ClusteringDiarizer
 from nemo.collections.asr.parts.utils.speaker_utils import (
+    audio_rttm_map,
     get_uniqname_from_filepath,
     labels_to_rttmfile,
     rttm_to_labels,
     write_rttm2manifest,
 )
-from nemo.collections.asr.parts.utils.streaming_utils import AudioFeatureIterator, FrameBatchASR
-from nemo.collections.common.tokenizers.tokenizer_spec import TokenizerSpec
 from nemo.utils import logging
+from nemo.utils.decorators.experimental import experimental
 
-__all__ = ['ASR_DIAR_OFFLINE']
+try:
+    import arpa
+except ImportError:
+    logging.info("arpa is not installed. You must install arpa to refine diarization result with LM.")
```

Comment:
You are checking for this twice.

Reply:
If I don't use try/except, then it will pop-out error if arpa is not installed. If you want to remove the warning sign, then it will make more sense.

Reply:
Need to discuss this.